spring-security-oauth2/src/test/java/org/springframework/security/oauth2/provider/approval/TokenStoreUserApprovalHandlerTests.java

```diff
@@ -75,6 +78,21 @@ public void testMemorizedApproval() {
 		assertTrue(handler.isApproved(authorizationRequest, userAuthentication));
 	}
 
+	@Test
+	public void testScopedApproval() {
```

Comment:
It's not clear to me whether this is an issue or not. Can you please add a test that demonstrates the issue without this change taking effect.